src/main.cpp

```diff
@@ -2542,9 +2547,9 @@ bool CheckBlock(const CBlock& block, CValidationState& state, bool fCheckPOW, bo
 
 bool ContextualCheckBlockHeader(const CBlockHeader& block, CValidationState& state, CBlockIndex * const pindexPrev)
 {
-    const Consensus::Params& consensusParams = Params().GetConsensus();
```

Comment:
In #5975, which was older than this and is double-linked to this one. In fact, the variable is being used by GetNextWorkRequired and isSuperMajority and I'm not getting that warning...Are you sure you sure you got that warning? Are you on master?